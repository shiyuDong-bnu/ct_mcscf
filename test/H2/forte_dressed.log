2025-03-07:16:59:05,935 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'forte__job_type': 'MCSCF_TWO_STEP', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-08, 'forte__d_convergence': 1e-08, 'forte__active_space_solver': 'FCI', 'forte__active': [4], 'forte__int_type': 'FCIDUMP', 'forte__fcidump_file': 'DRESSED.DUMP', 'forte__r_convergence': 1e-08, 'forte__mcscf_g_convergence': 1e-06, 'forte__mcscf_micro_maxiter': 4, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ', 'function_kwargs': {}}
2025-03-07:16:59:05,937 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz, molecule=CO, nre=0.52917721067
