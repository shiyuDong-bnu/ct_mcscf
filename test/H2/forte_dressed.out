
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {HEAD} f53cdd7 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Friday, 07 March 2025 04:59PM

    Process ID: 376657
    Host:       debianlab
    PSIDATADIR: /home/sydong/install_tutorial/psi4/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
import forte

molecule CO{
  0 1
  H 
  H  1 1.0
  symmetry c1
}

set {
  basis                 cc-pvdz
  reference             rhf
  scf_type              pk
  maxiter               300
  e_convergence         10
  d_convergence         8
#  docc                  [5,0,1,1]
}

set forte {
  job_type              mcscf_two_step
  active_space_solver                fci
#  restricted_docc       [2]
  active                [4,]
  int_type              fcidump
  fcidump_file          DRESSED.DUMP 
  e_convergence         8  # energy convergence of the FCI iterations
  r_convergence         8  # residual convergence of the FCI iterations
  mcscf_e_convergence  8  # energy convergence of the MCSCF iterations
  mcscf_g_convergence  6  # gradient convergence of the MCSCF iterations
  mcscf_micro_maxiter  4  # do at least 4 micro iterations per macro iteration
}

Eforte = energy('forte')
--------------------------------------------------------------------------

Scratch directory: /home/sydong/scratch/psi4/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch: HEAD - git commit: 259d7b80

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

  Forte will use custom integrals
  Preparing forte objects from a custom source

  Reading integral information from FCIDUMP file DRESSED.DUMP

  ==> MO Space Information <==

  -------------------------------
                        A   Sum
  -------------------------------
    FROZEN_DOCC         0     0
    RESTRICTED_DOCC     0     0
    GAS1                4     4
    GAS2                0     0
    GAS3                0     0
    GAS4                0     0
    GAS5                0     0
    GAS6                0     0
    RESTRICTED_UOCC     6     6
    FROZEN_UOCC         0     0
    Total              10    10
  -------------------------------

  ==> Integral Transformation <==

  Number of molecular orbitals:                         10
  Number of correlated molecular orbitals:              10
  Number of frozen occupied orbitals:                    0
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                       Custom


  Timing for freezing core and virtual orbitals:              0.000 s.
  Timing for preparing custom integrals:                      0.000 s.

          -----------------------------------------------------------
                  Multi-Configurational Self Consistent Field
                Two-Step Approximate Second-Order AO Algorithm
            written by Chenyang Li, Kevin P. Hannon, and Shuhe Wang
          -----------------------------------------------------------


  ==> MCSCF Calculation Information <==

    --------------------------------------------------------
    Print level                                      Default
    Integral type                                    FCIDUMP
    CI solver type                                       FCI
    Final orbital type                             CANONICAL
    Derivative type                                     NONE
    Optimize orbitals                                   TRUE
    Include internal rotations                         FALSE
    Debug printing                                     FALSE
    Energy convergence                             1.000e-08
    Gradient convergence                           1.000e-06
    Max value for rotation                         2.000e-01
    Max number of macro iter.                            100
    Max number of micro iter. for orbitals                 4
    Max number of micro iter. for CI                      12
    DIIS start                                            15
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    Frequency of DIIS extrapolation                        1
    --------------------------------------------------------

  ==> Independent Orbital Rotations <==

    ORBITAL SPACES                         A
    ----------------------------------------
             ACTIVE / RESTRICTED_DOCC      0
    RESTRICTED_UOCC /          ACTIVE     24
    RESTRICTED_UOCC / RESTRICTED_DOCC      0
    ----------------------------------------

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              1
    number of beta electrons                               1
    number of alpha strings                                4
    number of beta strings                                 4
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                                16
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------

  ==> Initial Guess <==

  Initial guess determinants:         16

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
    10       1       *
     6       3        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0       -1.132766654794  +0.000000  added
  -------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-08
    Residual convergence threshold                 1.000e-08
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                     16
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: adding 1 guess vectors
  Iteration     Average Energy            max(âˆ†E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0       -1.132766654794        1.132766654794        0.000000000000      1
       1       -1.132766654794        0.000000000000        0.000000000000      2
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    2000     -0.98770104
    0200      0.10553016
    0a0b     -0.06594802
    0b0a     -0.06594802
    0002      0.04997433
    0020      0.04445854

    Total Energy:      -1.132766654794, <S^2>: 0.000000
    Time for FCI:       0.000790216000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0       -1.132766654794   0.000000
    --------------------------------------------------------

  ==> Natural Orbitals <==

        1A      1.951370      2A      0.044588      3A      0.003965  
        4A      0.000077  
sydong:  unsymmetrize 2RDM
